python/pyspark/ml/recommendation.py

```diff
@@ -332,6 +338,20 @@ def getFinalStorageLevel(self):
         """
         return self.getOrDefault(self.finalStorageLevel)
 
+    @since("2.0.0")
```

Comment:
Super minor - but we probably want to bump the since annotations here (looks like you already got the Scala ones).